mintpy/prep_cosicorr.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python3
+############################################################
+# Program is part of MintPy                                #
+# Copyright (c) 2013, Zhang Yunjun, Heresh Fattahi         #
+# Author: Forrest Williams, Mar 2021                       #
+############################################################
+
+
+import os
+import sys
+import argparse
+from datetime import datetime
+import numpy as np
+from mintpy.utils import readfile, writefile, utils as ut
+
+
+#########################################################################
+EXAMPLE = """example:
+  prep_cosicorr.py  offsets/*offset.tif -m metadata.txt
+  prep_cosicorr.py  snr/*snr.tif -m metadata.txt
+"""
```

Comment:
I would put an example metadata.txt file here for easy reference.

Reply:
Thanks, I'll add this